src/components/Topnav.vue

```diff
@@ -123,6 +124,7 @@ onMounted(() => setTitle());
               v-if="!domain"
               @click="toggleSkin"
               class="float-right ml-2"
+              aria-label="Toggle Skin"
```

Comment:
This should use a translation message, and we should use "Sentence case" instead of "Title Case", (Toggle skin)

Reply:
@bonustrack Just updated it, I'm not sure, should I add it only to the `default.json` file or all files in locales?

Reply:
@bigint just he default.json